functions/src/google/places.ts

```diff
@@ -0,0 +1,69 @@
+import * as functions from 'firebase-functions';
+import * as request from 'request-promise-native';
+import { v4 } from 'uuid';
+
+import { GooglePlaceFind, GooglePlaceSearch } from '../models';
+import { CallableContext } from 'firebase-functions/lib/providers/https';
+
+export const GooglePlacesSearch = functions.https.onCall(async (data: any, context: CallableContext): Promise<any> => {
+    const uid = data.sessionId ? data.sessionId : v4();
+    const value = data.placeName;
+    const type = data.type;
+
+    const method = 'POST';
+    const uri = `https://maps.googleapis.com/maps/api/place/autocomplete/json`;
```

Comment:
This could just be single quotes

Reply:
ah yea since things have changed here I can fix this